node/nodeRunner.go

```diff
@@ -516,9 +516,13 @@ func (nr *nodeRunner) executeOneComponentCreationCycle(
 
 	log.Info("application is now running")
 
+	delayInSecBeforeAllowingVmQueries := configs.GeneralConfig.WebServerAntiflood.VmQueryDelayAfterStartInSec
+	if delayInSecBeforeAllowingVmQueries == 0 {
```

Comment:
Shall we add a warn here to specify that we override the config value?

Reply:
done